data_structures/hashing/number_theory/chinese_remainder_theorem.py

```diff
@@ -0,0 +1,70 @@
+# Chinese Remainder Theorem:
+
+# If GCD(a,b) = 1, then for any remainder ra modulo a and any remainder rb modulo b there exists integer n,
```

Comment:
Please expand the acronym GCD and try to keep the lines to 88 characters or less.